rds-instance-full/main.tf

```diff
@@ -40,6 +40,29 @@ module "rds_security_group" {
   ingress_rules       = ["${var.ingress_rule}"]
 }
 
+module "custom_security_group" {
+  source = "terraform-aws-modules/security-group/aws"
+
+  create = "${var.rds_sg == "" ? 0 : 1 }"
+
+  name        = "${local.identifier}-rds-ecs"
+  description = "Security group with RDS ports open for ECS"
+  vpc_id      = "${data.terraform_remote_state.vpc.vpc_id}"
+
+  ingress_cidr_blocks = ["${data.terraform_remote_state.vpc.vpc_cidr_block}"]
+
+  ingress_with_source_security_group_id = [
+    {
+      rule                     = "${var.ingress_rule}"
+      source_security_group_id = "${var.rds_sg}"
```

Comment:
`rds_sg` => `custom_sg_id`

Reply:
Naming this is the biggest problem of IT, thanks!